modules/oadp-ceph-preparing-cephfs-crs.adoc

```diff
@@ -17,23 +17,21 @@ When you install {rh-storage-first}, it automatically creates a default CephFS `
 [source,yaml]
 ----
 apiVersion: snapshot.storage.k8s.io/v1
```

Comment:
DeletionPolicy is a required field without specifying it we cannot create this CR.  Here I think we should mention that OADP supports both Retain and Delete as deletion policy. 
```
oc explain vsclass
   deletionPolicy	<string> -required-
     deletionPolicy determines whether a VolumeSnapshotContent created through
     the VolumeSnapshotClass should be deleted when its bound VolumeSnapshot is
     deleted. Supported values are "Retain" and "Delete". "Retain" means that
     the VolumeSnapshotContent and its physical snapshot on underlying storage
     system are kept. "Delete" means that the VolumeSnapshotContent and its
     physical snapshot on underlying storage system are deleted. Required.
```

Reply:
If I remember correctly OADP 1.2 dataMover only works with deletionPolicy delete

Reply:
@PrasadJoshi12 
I updated the file based on your comments.

Reply:
@PrasadJoshi12 I believe the issue you are referring to OADP-1.2 was fixed by @sseago .  Velero in 1.2 and 1.3 latest releases should patch object as needed w/ retain or delete.  Perhaps the real engineers ( not me ) can get this settled.  @sseago  I have failed to clear up the confusion here, perhaps you can please lend me a hand.

Reply:
@weshayutin That's right -- didn't you verify last week that the policy didn't matter for 1.2 datamover as well? So "retain or delete" should be valid across OADP versions.

Reply:
@sseago 
I'm still a little confused. Prasad is on PTO till next week.
Could you please look over the PR and point out if I need to make any changes?
Thanks,
Carmi

Reply:
@CarmiWisemon There are a couple places here where OADP 1.2 is called out as having more specific requirements for deletion policy. That is not correct -- for both 1.2 and 1.3, we support both retain and delete, so we can remove the 1.2-specific comments here.

Reply:
@sseago I think we have not fixed this issue for 1.2, we fixed it as doc bug. 
https://issues.redhat.com/browse/OADP-1681

Reply:
@PrasadJoshi12 There was some confusion on this point, so @weshayutin tested it out -- Wes, didn't you determine that we had fixed the issue in 1.2? Or am I the one misremembeirng now?

Reply:
Working w/ @PrasadJoshi12 we both agree and we should be very explicit in the doc.
A.  OADP-1.2.x datamover b/r requires the volume snapshot class to be set to deletionPolicy=Retain
B. OADP-1.2.x CSI b/r vsc class can be either deletionPolicy=[Retain,Delete]
C. OADP-1.3.x  for ANY backup type can be deletionPolicy=[Retain, Delete]

Also noting OADP-1.2.x DM backups / restore are tech preview and not supported w/ a support exception.

Reply:
I agree.

Reply:
@sseago FYI ^

Reply:
@weshayutin @PrasadJoshi12 
1. I updated the OADP 1.2 DM section based on your summary.
2. I added two separate Deletion Policy sections for OADP 1.2 and 1.3.
3. Please review the changes.